lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/MultiRuntimeYarnBootstrapDriverConfigGenerator.java

```diff
@@ -0,0 +1,255 @@
+                                                         final ConfigurationSerializer configurationSerializer) {
+    this.configurationSerializer = configurationSerializer;
+    this.reefFileNames = reefFileNames;
+  }
+
+
+  private void createYarnRuntimeDefinition(
+          final AvroYarnJobSubmissionParameters yarnJobSubmissionParams,
+          final AvroJobSubmissionParameters jobSubmissionParameters,
+          final MultiRuntimeDefinitionBuilder builder) {
+    // create and serialize yarn configuration if defined
+    final Configuration yarnDriverConfiguration = YarnDriverConfiguration.CONF
+            .set(YarnDriverConfiguration.JOB_SUBMISSION_DIRECTORY,
+                    yarnJobSubmissionParams.getDfsJobSubmissionFolder().toString())
+            .set(YarnDriverConfiguration.JOB_IDENTIFIER,
+                    jobSubmissionParameters.getJobId().toString())
+            .set(YarnDriverConfiguration.CLIENT_REMOTE_IDENTIFIER,
+                    ClientRemoteIdentifier.NONE)
+            .set(YarnDriverConfiguration.JVM_HEAP_SLACK, 0.0)
+            .set(YarnDriverConfiguration.RUNTIME_NAMES, RuntimeIdentifier.RUNTIME_NAME)
+            .build();
+
+    // add yarn runtime to the builder
+    builder.addRuntime(yarnDriverConfiguration, RuntimeIdentifier.RUNTIME_NAME);
+  }
+
+  private void createLocalRuntimeDefinition(
+          final AvroMultiRuntimeAppSubmissionParameters multiruntimeAppSubmissionParams,
+          final AvroJobSubmissionParameters jobSubmissionParameters,
+          final MultiRuntimeDefinitionBuilder builder) {
+    // create and serialize local configuration if defined
+    final Configuration localModule = LocalDriverConfiguration.CONF
+            .set(LocalDriverConfiguration.MAX_NUMBER_OF_EVALUATORS, multiruntimeAppSubmissionParams
+                    .getLocalRuntimeAppParameters().getMaxNumberOfConcurrentEvaluators())
+            // ROOT FOLDER will point to the current runtime directory
+            .set(LocalDriverConfiguration.ROOT_FOLDER, ".")
+            .set(LocalDriverConfiguration.JVM_HEAP_SLACK, 0.0)
+            .set(LocalDriverConfiguration.CLIENT_REMOTE_IDENTIFIER, ClientRemoteIdentifier.NONE)
+            .set(LocalDriverConfiguration.JOB_IDENTIFIER,
+                    jobSubmissionParameters.getJobId().toString())
+            .set(LocalDriverConfiguration.RUNTIME_NAMES,
+                    org.apache.reef.runtime.local.driver.RuntimeIdentifier.RUNTIME_NAME)
+            .build();
+
+    // add local runtime to the builder
+    builder.addRuntime(localModule, org.apache.reef.runtime.local.driver.RuntimeIdentifier.RUNTIME_NAME);
+  }
+
+  public String writeDriverConfigurationFile(final String bootstrapJobArgsLocation,
+                                             final String bootstrapAppArgsLocation) throws IOException {
+    final File bootstrapJobArgsFile = new File(bootstrapJobArgsLocation).getCanonicalFile();
```

Comment:
I may have wrote the code for that. In that case, that was a mistake on my part. I don't think there is a need for it here.